modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPkgFilterToSQL.java

```diff
@@ -237,4 +238,11 @@ protected Object visitBinarySpatialOperator(
 
         return extraData;
     }
+
+    @Override
+    protected FilterCapabilities createFilterCapabilities() {
```

Comment:
This is worrysome, no clue as to why FilterToSQL does not claim support for PropertyIsLike, looks like an issue that might be affecting other databases.. I've done a quick check, PostGIS and Oracle setup the filter capabilities in a different way and end up having PropertyIsLike support, but for example MySQL is affected by this problem as well.

@ydeliorman could you try declaring support for PropertyIsLike in the base class instead, and see if any test breaks?

Reply:
@aaime after moving the logic should I be running the tests on the GeoTools side? Do I need to get a full run at GeoServer sider too?

Reply:
Running all database tests on your machine is quite a bit of setup, but the PR checks do it for you so... just make the change and push it here, then see what the various database test runs tell you, e.g.:

![image](https://user-images.githubusercontent.com/325433/149926273-144b7ac3-521a-45d7-a6a9-4673915ff5e6.png)